packages/tests/tests/FRAMEWORK+SERVER+RPC+drizzle.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { describeBati, exec, npmCli } from "@batijs/tests-utils";
+
+export const matrix = [
+  ["solid", "react", "vue"],
+  ["express", "h3", "hono", "fastify"],
+  ["trpc", "telefunc"],
+  "drizzle",
+  "eslint",
```

Comment:
This will also test when neither `trpc` nor `telefunc` are present.
```suggestion
  ["solid", "react", "vue"],
  ["express", "h3", "hono", "fastify", "hattip"],
  ["trpc", "telefunc", undefined],
  "drizzle",
  "eslint",
```